symengine/lib/symengine_wrapper.pyx

```diff
@@ -2078,22 +2081,134 @@ class DenseMatrixIter(object):
+        for i in range(o.rows):
+            for j in range(o.cols):
+                result[i, j + self.cols] = o[i, j]
+        return result
+
+    def col_join(self, bott):
+        cdef DenseMatrixBase o = _sympify(bott)
+        if self.cols != o.cols:
+            raise ShapeError("`self` and `rhs` must have the same number of columns.")
+        cdef DenseMatrixBase result = zeros(self.rows + o.rows, self.cols)
+        for i in range(self.rows):
+            for j in range(self.cols):
+                result[i, j] = self[i, j]
+        for i in range(o.rows):
+            for j in range(o.cols):
+                result[i + self.rows, j] = o[i, j]
+        return result
+
+    def col_del(self, i):
+        if i < -self.cols or i >= self.cols:
+            raise IndexError("Index out of range: 'i=%s', valid -%s <= i < %s"
+                             % (i, self.cols, self.cols))
+        cdef DenseMatrixBase result = zeros(self.rows, self.cols -1)
+        for j in range(self.rows):
+            for k in range(self.cols - 1):
+                if k >= i:
+                    result[j, k] = self[j, k + 1]
+                else:
+                    result[j, k] = self[j, k]
+        return result
+
+    def col_op(self, j, f):
+        raise NotImplementedError
+
+    def col_swap(self, i, j):
+        for k in range(0, self.rows):
+            self[k, i], self[k, j] = self[k, j], self[k, i]
+
+    def copyin_list(self, key, value):
+        raise NotImplementedError
+
+    def copyin_matrix(self, key, value):
+        rlo, rhi, clo, chi = self.key2bounds(key)
+        shape = value.shape
+        dr, dc = rhi - rlo, chi - clo
+        if shape != (dr, dc):
+            raise ShapeError("The Matrix `value` doesn't have the same"\
+                             "dimensions as the in sub-Matrix given by `key`.")
+
+        for i in range(value.rows):
+            for j in range(value.cols):
+                self[i + rlo, j + clo] = value[i, j]
+
+    def fill(self, value):
+        for i in range(self.rows):
+            for j in range(self.cols):
+                self[i, j] = value
+
+    def row_del(self, i):
+        if i < -self.rows or i >= self.rows:
+            raise IndexError("Index out of range: 'i = %s', valid -%s <= i"
+                             " < %s" % (i, self.rows, self.rows))
+        if i < 0:
+            i += self.rows
+        cdef DenseMatrixBase result = zeros(self.rows - 1, self.cols)
+        for j in range(self.rows - 1):
+            for k in range(self.cols):
+                if j >= i:
+                    result[j, k] = self[j + 1, k]
+                else:
+                    result[j, k] = self[j, k]
+        return result
+
+    def row_op(self, i, f):
+        raise NotImplementedError
+
+    def row_swap(self, i, j):
+        for k in range(0, self.cols):
+            self[i, k], self[j, k] = self[j, k], self[i, k]
+
+    def simplify(self, *args):
+        raise NotImplementedError
```

Comment:
Can you delete these not implemented classes? This would break some existing code I have if this is defined but not implemented